app/Server.php

```diff
@@ -243,15 +262,87 @@ public function updateUsage()
                     $meeting->room->voice_participant_count = $meetingStat->voice_participant_count = $bbbMeeting->getVoiceParticipantCount();
                     $meeting->room->video_count             = $meetingStat->video_count             = $bbbMeeting->getVideoCount();
 
-                    if (setting('log_attendance')) {
```

Comment:
Shouldn't the live meeting status (see the lines above) also be empty is the attendance statistics disabled?

Reply:
No, we need this data to show correct results in the currently running meetings view

Reply:
The option should only disable archiving this data